packages/forms-web-app/src/routes/file-based-router/comment-planning-appeal/appeals/_appealNumber/index.njk

```diff
@@ -40,7 +40,7 @@
 			{% if appeal.status == 'open' %}
 				{{ govukButton({
 					text: "Comment on this appeal",
-					href: "#",
+					href: "../your-name",
```

Comment:
does `../` work like this in nunjucks?? 👀 big if true

Reply:
It works in practice, which I must admit was a little surprising